_P208_Nokia_LCD_5110.ino

```diff
@@ -217,32 +219,37 @@ boolean Plugin_208(byte function, struct EventStruct *event, String& string){
             argIndex = string.lastIndexOf(',');
             tmpString = string.substring(argIndex + 1);
             if (tmpString.equalsIgnoreCase(F("Clear"))){
-              addLog(LOG_LEVEL_INFO, F("Clear Display"));
+              myLog("Clear Display");
               lcd3->clearDisplay();
               lcd3->display();
             }
             if (tmpString.equalsIgnoreCase(F("blOn"))){
               success = true;
               PCONFIG(0) = 1;
-              setBacklight(event); 
+              setBacklight(event);
             }
             if (tmpString.equalsIgnoreCase(F("blOff"))){
               success = true;
               PCONFIG(0) = 0;
-              setBacklight(event); 
+              setBacklight(event);
             }
-          break;
-        }
+            break;
+          }
         }
       }
   }  // switch (function)
   return success;
 }
 
+void myLog (String message) {
+  addLog(LOG_LEVEL_INFO, "P208: " + message);

```

Comment:
That's quite inefficient, should be something like:
```
void myLog(const String & message) {
  String log;
  log.reserve(message.length() + 6);
  log += F("P208: ");
  log += message;
  addLogMove(LOG_LEVEL_INFO, log);
}
```

Reply:
I'm using the "ESPEasy_ESP32_mega-20211224" the latest version, but I have the compile error "addLogMove was not declared". So I will use your code but still use "addlog"......

Reply:
That feature/optimization `addLogMove` was introduced after the 20211224 release, correct, so `addLog` is the best alternative for now.
When developing on ESPEasy, it is advised to use the `mega` brach from the regular ESPEasy repository, that way you also help in beta-testing, and improving, the code, and always have the latest improvements available 😃